### PR TITLE
fix: Update api-access page to use get_userlist() #6063

### PR DESCRIPTION
**Pull request by laf**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

Fixes: #6063

### CODE REVIEW

**Comment by laf** [comment]
Confirmed working in the issue.

File: html/pages/api-access.inc.php [hunk 1]
```diff
@@ -136,15 +136,21 @@
         </tr>
 ';
 
-foreach (dbFetchRows('SELECT `AT`.*,`U`.`username` FROM `api_tokens` AS AT JOIN users AS U ON AT.user_id=U.user_id ORDER BY AT.user_id') as $api) {
+$tmp_users = get_userlist();
```
Comment (murrant) [inline html/pages/api-access.inc.php]: Why call get_userlist() again when we already got that data a little bit ago?

File: html/pages/api-access.inc.php [hunk 2]
```diff
@@ -136,15 +136,21 @@
...
+foreach (dbFetchRows('SELECT * FROM `api_tokens` ORDER BY user_id') as $api) {
     if ($api['disabled'] == '1') {
         $api_disabled = 'checked';
     } else {
         $api_disabled = '';
     }
+    foreach ($tmp_users as $tmp_user) {
+        if ($tmp_user['user_id'] === $api['user_id']) {
+            $user_details = $tmp_user;
```
Comment (murrant) [inline html/pages/api-access.inc.php]: A `break;` here would probably be a good idea.

**Comment by murrant** [comment]
Any thoughts on how to fix the incorrect user ids for ldap auth?  changes in the auth?

**Comment by laf** [comment]
ldap auth get_userlist() and get_userid() looks like it gets those values from ldap itself so we would then insert that into the db. I'm not sure why you'd get different IDs unless LDAP itself does that.

**Comment by murrant** [comment]
Hmm, likely because my ldap and sql username matches...

**Comment by murrant** [comment]
OK, well this bug is in the ldap auth, so I think this is ok.

**Pull request merged**